theano/tensor/basic.py

```diff
@@ -2168,6 +2168,67 @@ def sqr(a):
 square = sqr
 
 
+def cov(X, y=None, rowvar=True, bias=False, ddof=None, fweights=None, aweights=None):
+    """Calculate the covariance matrix.
+    Covariance indicates the level to which two variables vary together.
+    If we examine N-dimensional samples, :math:`X = [x_1, x_2, ... x_N]^T`,
+    then the covariance matrix element :math:`C_{ij}` is the covariance of
+    :math:`x_i` and :math:`x_j`. The element :math:`C_{ii}` is the variance
+    of :math:`x_i`. Code and docstring ported from numpy.
+    ----------
+    a : array_like
```

Comment:
We try to mimic the numpy API in general so going with what numpy does is not a bad idea.  In any case, this absolutely needs to have the same name in the function definition and the doc.